app/controllers/users_controller.rb

```diff
@@ -6,5 +6,6 @@ def show
       :order_direction => 'desc',
       :per_page => 20
     )
+    @latest_access_token = @user.access_tokens.last&.token
```

Comment:
Userクラスに最新のアクセストークンを返すメソッドがあるとかっこいいです。

```suggestion
    @latest_access_token = @user.latest_access_token
```

のイメージです。

Reply:
修正しました。
https://github.com/pubannotation/pubdictionaries/pull/142/commits/6ae5df327d38ca9018b96f195474cdbee974028c